whoville/mayor.py

```diff
@@ -139,8 +139,9 @@ def step_2_init_infra(create_wait=0):
         create=True,
         purge=horton.global_purge
     )
-    log.info("Ensuring Environment Credential for Director")
-    horton.cadcred = director.get_environment()
+    if config.profile['platform']['provider'] == 'EC2':
```

Comment:
It seems like this would prevent Director being properly initialised unless running on EC2, did you test Director with this PR?

Reply:
director.py itself will exit with an error unless it's EC2, so here I'm skipping it altogether.  

I can work on Director configuration for GCP, but it wasn't a priority for me.  It will start, but I've got to look into pushing the credential to Director.  I intend to do that in a separate PR.